dhctl/pkg/operations/bootstrap/cluster-bootstrapper.go

```diff
@@ -319,13 +285,9 @@ func (b *ClusterBootstrapper) Bootstrap() error {
 			deckhouseInstallConfig.TerraformState = baseOutputs.TerraformState
 
 			if baseOutputs.BastionHost != "" {
-				setBastionHost(baseOutputs.BastionHost, sshClient)
```

Comment:
This func also sets sshClient.Settings.BastionHost|Port|User params. We should not forget to initialize those in the outside code where sshClient is created.